web/docs/project/custom-vite-config.md

```diff
@@ -111,3 +111,58 @@ If you, for example, want to serve the client from a different path than `/`, yo
     ```
   </TabItem>
 </Tabs>
+
+## Chrome DevTools automatic workspace mapping {#devtools-workspace}
+
+Want edits you make in Chrome DevTools Sources panel to save directly to your project files? You can enable [Chrome DevTools Workspace](https://developer.chrome.com/docs/devtools/workspace/) support in development with the community plugin **vite-plugin-devtools-json** (no Wasp fork required).
+
+1. Install the plugin as a **dev-dependency**:
+
+```bash
+npm add -D vite-plugin-devtools-json
```

Comment:
When using `npm`, this command is usually done with `i` or `install` instead of `add`. And everywhere else in our docs we use `i`, so let's change it.

```suggestion
npm i -D vite-plugin-devtools-json
```